mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb

```diff
@@ -5998,6 +6050,17 @@ func flattenGatewayApiConfig(c *container.GatewayAPIConfig) []map[string]interfa
 	}
 }
 
+func flattenClusterNetworkPerformanceConfig(c *container.ClusterNetworkPerformanceConfig) []map[string]interface{} {
+	if c == nil {
+		return nil
+	}
+	return []map[string]interface{}{
+		{
+			"totalEgressBandwidthTier":        c.Tier,
```

Comment:
This line corresponds to the below line of the error I posted, and believe it's because the field needs to be renamed `TotalEgressBandwidthTier`

```
google-beta/services/container/resource_container_cluster.go:5951:34: c.Tier undefined (type *"google.golang.org/api/container/v1beta1".ClusterNetworkPerformanceConfig has no field or method Tier)
```